### ISSUE TITLE
`definePageMeta` not working with `nuxt-seo-kit`

### ISSUE THREAD

**Issue by mukundshah** [victorgarciaesgi/nuxt-typed-router#79]
I am using `nuxt-seo-kit` and it relies on `title` and `description` set through `definePageMeta`, but now, `nuxt-typed-router` throws errors all over the place.

This is my current usage:

```ts
definePageMeta({
  title: 'Home',
  description: 'This is homepage'
});
```

Error:
```
No overload matches this call.
  The last overload gave the following error.ts(2769)
__definePageMeta.ts(92, 17): The last overload is declared here.
__definePageMeta.ts(127, 17): The call would have succeeded against this implementation, but implementation signatures of overloads are not externally visible.
```

**Comment by victorgarciaesgi** [comment]
Fixed in `v3.0.6`

**Comment by mukundshah** [comment]
I came here to update the title, but you already did it.

Thank you! x2
